pkg/operator/status/status_controller.go

```diff
@@ -217,10 +220,23 @@ func (c StatusSyncer) Sync(ctx context.Context, syncCtx factory.SyncContext) err
 	if _, updateErr := c.clusterOperatorClient.ClusterOperators().UpdateStatus(ctx, clusterOperatorObj, metav1.UpdateOptions{}); updateErr != nil {
 		return updateErr
 	}
-	syncCtx.Recorder().Eventf("OperatorStatusChanged", "Status for clusteroperator/%s changed: %s", c.clusterOperatorName, configv1helpers.GetStatusDiff(originalClusterOperatorObj.Status, clusterOperatorObj.Status))
+	if !skipOperatorStatusChangedEvent(originalClusterOperatorObj.Status, clusterOperatorObj.Status) {
+		syncCtx.Recorder().Eventf("OperatorStatusChanged", "Status for clusteroperator/%s changed: %s", c.clusterOperatorName, configv1helpers.GetStatusDiff(originalClusterOperatorObj.Status, clusterOperatorObj.Status))
+	}
 	return nil
 }
 
+func skipOperatorStatusChangedEvent(originalStatus, newStatus configv1.ClusterOperatorStatus) bool {
+	originalCopy := *originalStatus.DeepCopy()
+	for i, condition := range originalCopy.Conditions {
+		switch condition.Type {
+		case configv1.OperatorAvailable, configv1.OperatorDegraded, configv1.OperatorProgressing, configv1.OperatorUpgradeable:
+			originalCopy.Conditions[i].Message = strings.TrimPrefix(condition.Message, "\ufeff")
```

Comment:
caps or not caps?  didn't I just review a change for caps?

Reply:
The change you might be referring to [this change](https://github.com/openshift/origin/pull/28438/files) where I had to switch from checking for the literal `NBSP` to a quoted version (e.g. the message was formatted with `%q` somewhere along the way).